{"name":"Python 报错","slug":"Python-报错","count":1,"postlist":[{"title":"ERROR:解决 python nltk punkt not found 问题","uid":"ba502fc9de400c4cec9ee8172833afd5","slug":"ERROR-解决-python-nltk-punkt-not-found-问题-zh","date":"2022-12-21T07:26:27.000Z","updated":"2022-12-22T04:31:50.821Z","comments":true,"path":"api/articles/ERROR-解决-python-nltk-punkt-not-found-问题-zh.json","keywords":null,"cover":null,"text":" Click to Switch to English&#x2F;Chinese 报错内容： LookupError: ********************************************************************** Resource ...","link":"","photos":[],"count_time":{"symbolsCount":"1.5k","symbolsTime":"1 mins."},"categories":[{"name":"Python 报错","slug":"Python-报错","count":1,"path":"api/categories/Python-报错.json"}],"tags":[{"name":"疑难杂症","slug":"疑难杂症","count":1,"path":"api/tags/疑难杂症.json"},{"name":"Python","slug":"Python","count":1,"path":"api/tags/Python.json"},{"name":"nltk","slug":"nltk","count":1,"path":"api/tags/nltk.json"},{"name":"punkt","slug":"punkt","count":1,"path":"api/tags/punkt.json"}],"author":{"name":"Geek Boom","slug":"blog-author","avatar":"https://g33kb00m.github.io/static/img/geekboomava.png","link":"/","description":"凝聚极客精神，如风暴般思维碰撞。<br /><b>Condensing the spirit of geeks,thoughts collide like a storm.</b>","socials":{"github":"","twitter":"https://twitter.com/G33kBoom","stackoverflow":"","wechat":"","qq":"","weibo":"https://m.weibo.cn/u/5697551873","zhihu":"","csdn":"https://blog.csdn.net/GeekBoom","juejin":"","customs":{"bilibili":{"icon":"https://www.geekboom.org/static/img/bilibili_icon.svg","link":"https://space.bilibili.com/1515037948"},"youtube":{"icon":"https://www.geekboom.org/static/img/youtube_icon.svg","link":"https://www.youtube.com/channel/UCqPgUhlSZEskxWFuPRQt2JQ"},"discord":{"icon":"https://www.geekboom.org/static/img/discord_icon.svg","link":"https://discord.com/"}}}},"feature":true}]}